packages/components/menu/src/menu-button.stories.ts

```diff
@@ -121,41 +132,42 @@ export const Disabled: Story = {
   }
 };
 
-export const SingleSelect: Story = {
+export const IconAndText: Story = {
   args: {
-    body: html`<span slot="button">View</span>`,
-    menuItems: () => html`
-      <sl-menu-item selectable selected shortcut="$mod+Digit1">
-        <sl-icon name="far-list"></sl-icon>
-        List
-      </sl-menu-item>
-      <sl-menu-item selectable shortcut="$mod+Digit2">
-        <sl-icon name="far-rectangles-mixed"></sl-icon>
-        Cards
-      </sl-menu-item>
-      <sl-menu-item selectable shortcut="$mod+Digit3">
-        <sl-icon name="far-table-cells"></sl-icon>
-        Grid
-      </sl-menu-item>
-    `,
-    selects: 'single'
+    ...Basic.args,
+    body: html`
+      <sl-icon name="far-gear" slot="button"></sl-icon>
+      <span slot="button">Settings</span>
+    `
   }
 };
 
-export const MultiSelect: Story = {
+export const Text: Story = {
   args: {
-    body: html`<span slot="button">Lists</span>`,
+    ...Basic.args,
+    body: html`<span slot="button">Settings</span>`
+  }
+};
+
+export const Submenu: Story = {
```

Comment:
In the example scrollbar is shown when I hover over the menu
<img width="380" alt="image" src="https://github.com/user-attachments/assets/db95413a-3c12-4250-9985-12cdb9995df5">

Reply:
Fixed